config/stack_ufs_weather_ci.yaml

```diff
@@ -211,7 +211,12 @@ grib_util:
   install_as: 1.2.2
   openmp: ON
 
+gsl:
+  build: YES
+  version: 2.6
+
```

Comment:
ufs-weather-model does not need gsl.  please remove.

Reply:
Not sure why these have been different from develop.  I did not change these.  Ah!  I believe this came from my merge of https://github.com/kgerheiser/hpc-stack/tree/feature/gsl into my branch.  
I changed it to NO.